addon/utils/get-asset-map-data.js

```diff
@@ -1,4 +1,10 @@
+import $ from 'jquery';
+
 export default function getAssetMapData() {
-  // This placeholder is replaced in the build step
-  return '__asset_map_placeholder__';
+  const assetMapString = $("meta[name='ember-cli-ifa:assetMap']").attr('content');
```

Comment:
In the interest of supporting [this RFC](https://github.com/emberjs/rfcs/blob/master/text/0294-optional-jquery.md), it might make more sense to use
`document.querySelector("meta[name='ember-cli-ifa:assetMap']").content` and not use jQuery in this file.

Reply:
Good call; thanks. Updated my PR.